storage/statedb/sync.go

```diff
@@ -338,3 +350,53 @@ func (s *TrieSync) commit(req *request) (err error) {
 	}
 	return nil
 }
+
+// PendingByDepth returns the pending trie count by given depth.
+func (s *TrieSync) PendingByDepth(depth int) int {
+	return s.pendingByDepth[depth]
+}
+
+// CommittedByDepth returns the committed trie count by given depth.
+func (s *TrieSync) CommittedByDepth(depth int) int {
+	return s.committedByDepth[depth]
+}
+
+// CalcProgressPercentage returns the progress percentage.
+func (s *TrieSync) CalcProgressPercentage() float64 {
+	var progress float64
+	//depth	max trie	resolution (%)
+	//0	 	1 	 		100.00000
+	//1	 	16 	 		6.25000
+	//2	 	256 	 	0.39063
+	//3	 	4,096 	 	0.02441
+	//4	 	65,536 	 	0.00153
+	//5	 	1,048,576 	0.00010
+
+	for i := 0; i < 6; i++ {
+		c, p := s.CommittedByDepth(i), s.PendingByDepth(i)
+		if p == 0 {
+			break
+		}
+		newProgress := float64(c) / float64(p)
+		if newProgress > progress {
+			progress = newProgress
+		}
+	}
+
+	for i := 0; i < 20; i++ {
+		var progressByDepth float64
+		c, p := s.CommittedByDepth(i), s.PendingByDepth(i)
+		if p == 0 {
+			break
+		}
+		if p > 0 {
+			progressByDepth = float64(c) / float64(p) * 100
+		}
+		logger.Info("Trie sync progress by depth #"+strconv.Itoa(i), "committed", c, "pending", p, "progress", progressByDepth)
```

Comment:
This log level will be changed to "debug" in future PR after developing State Migration.